### PR TITLE
0.4.1 release tweaks

### PR DESCRIPTION
**Pull request by matzew**
as per title: adding openshift specifics to the release-04 branch

### CODE REVIEW

**Comment by matzew** [comment]
@alanfx @mgencur this seems to be some hick-up...

as @markusthoemmes is seeing this today as well on the `-serving` repo PR

**Comment by bbrowning** [comment]
It's not picking up the right `INTERNAL_REGISTRY` value from https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-0.4-presubmits.yaml#L36-L37 in this job for whatever reason. Instead, it's trying to use the old OpenShift 3 image registry location. We should figure out why that env variable isn't getting passed here and perhaps also update our script to just change the default?

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -195,24 +256,32 @@ function run_e2e_tests(){
...
-  oc delete --ignore-not-found=true -f ${KNATIVE_ISTIO_CRD_YAML}
-  oc delete --ignore-not-found=true -f ${KNATIVE_ISTIO_YAML}
+  oc delete --ignore-not-found=true -f $ISTIO_RELEASE
+  oc delete --ignore-not-found=true -f $ISTIO_CRD_RELEASE
 }
 
 function delete_serving_openshift() {
   echo ">> Bringing down Serving"
-  oc delete --ignore-not-found=true -f ${KNATIVE_SERVING_RELEASE}
+  oc delete --ignore-not-found=true -f $SERVING_RELEASE
```
Comment (mgencur) [inline openshift/e2e-tests-openshift.sh]: I think this should be deleting the `serving-resolved.yaml` file. AFAIK, $SERVING_RESOLVED is not defined. We have the same problem in knative-serving.

**Pull request closed without merging**